src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -323,6 +324,17 @@ export class CreateScenariosComponent implements OnInit {
   get constrainsForm() {
     return this.forms.get('constrains');
   }
+
+  get showTreatmentsTab() {
+    return (
+      this.scenarioState === 'SUCCESS' &&
+      this.featureService.isFeatureEnabled('treatments')
```

Comment:
just curious if there's a benefit to putting the featureflag here, or just that it makes it easier to deal with the *NgIf / *appFeatureFlag collision?

Reply:
yeah just that, be able to combine two conditions for my ngif